lib/openapi_parser/schemas/base.rb

```diff
@@ -6,6 +6,9 @@ class Base
 
     attr_reader :parent, :raw_schema, :object_reference, :root
 
+    # Parent allOf reference, if schema is nested in allOf
+    attr_accessor :parent_all_of
```

Comment:
I think this code is unnecessary, so please delete it.

Reply:
ah, right, I forgot it there

Reply:
ok removed